templates/hooks/webacl/sdk_read_one_post_set_output.go.tpl

```diff
@@ -4,4 +4,32 @@
 	if err := rm.setOutputRulesNestedStatements(ko.Spec.Rules, resp); err != nil {
 		return nil, err
 	}
+	
+	
+	// If we have a WebACL ARN, fetch its logging configuration
```

Comment:
i think this webhook file could become one function call that embeds all `setOutputRulesNestedStatements` and `setLoggingConfg` etc.. in some other controller it's usually called `setResourceAdditionalFields`